src/test/java/org/fundacionjala/coding/yerel/DigitalRootTest.java

```diff
@@ -0,0 +1,51 @@
+package org.fundacionjala.coding.yerel;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * test of DigitalRootTest.
+ */
+public class DigitalRootTest {
+    private static final int NUMBER_SEVEN = 7;
+    private static final int NUMBER_SIX = 6;
+    private static final int NUMBER_TWO = 2;
+    private static final int NUMBER_16 = 16;
+    private static final int NUMBER_942 = 942;
+    private static final int NUMBER_493193 = 493193;
```

Comment:
Don't create constants
Please create final local variables in the method as actualResult and expectedResult